src/main/java/com/javatechie/springboot/crud/example/entity/Product.java

```diff
@@ -22,5 +23,48 @@
     private int id;
     private String name;
     private int quantity;
+    private int status;
     private double price;
-}
+    private int categoria;
+
+    public static Product mapProductDtoToProduct(ProductDto productDto) throws Exception {
```

Comment:
**Product** java class should be a POJO (representation of only fields with getters/setter). please move this method to another class, for example ProductTransformer,java